### ISSUE TITLE
creating cluster with workloadIdentity enabled is not supported as the workloadIdentity field is not part of the proto

### ISSUE THREAD

**Issue by nimrodf1** [googleapis/python-container#42]
If you are still having issues, please be sure to include as much information as possible:

#### Environment details

  - OS type and version:
  - Python version: 3.8.3
  - pip version: `pip --version`
  - `google-cloud-container` version: `pip show google-cloud-container`

#### Steps to reproduce

[Here](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.zones.clusters) is the description of the cluster resource. 
[Here](https://github.com/googleapis/python-container/blob/master/google/cloud/container_v1/proto/cluster_service.proto) is the proto for the cluster. As you can see it doesn't include the workloadIdentityConfig. 

#### Code example

This is a simplified version of the code I ran to find the issue - 
```python
from google.oauth2 import service_account
from google.cloud import container_v1beta1

class GoogleCloudKubernetesClient(object):

  def __init__(self, file, project_id, zone):
    credentials = service_account.Credentials.from_service_account_file(
        filename=file)
    self.client = container_v1beta1.ClusterManagerClient(credentials=credentials)
    self.project_id = project_id
    self.zone = zone

  def create_cluster(self, cluster):
    response = self.client.create_cluster(self.project_id, self.zone, cluster=cluster)


def main():
  cluster_data = {
      "name": "test-cluster",
      "master_auth": {
          "username": "admin",
      },
      "node_pools": [
          {"name": "asdf"}
          ],
      "initial_cluster_version": "1.16",
      "location": "europe-west1-b",
      "workload_identity_config": {"workload_pool": "velos-auto-1.svc.id.goog"}
  }

  kube = GoogleCloudKubernetesClient(file='~/service-account.json', project_id='proj-name', zone='europe-west1-b')

  kube.create_cluster(cluster_data)


if __name__ == '__main__':
  main()
```

#### Stack trace
```
Traceback (most recent call last):
  File "/usr/local/google/home/user/IdeaProjects/Go_tmp_tests/mian.py", line 45, in <module>
    main()
  File "/usr/local/google/home/user/IdeaProjects/Go_tmp_tests/mian.py", line 41, in main
    kube.create_cluster(cluster_data)
  File "/usr/local/google/home/user/IdeaProjects/Go_tmp_tests/mian.py", line 17, in create_cluster
    response = self.client.create_cluster(self.project_id, self.zone, cluster=cluster)
  File "/usr/local/google/home/niuser/.local/lib/python3.7/site-packages/google/cloud/container_v1beta1/gapic/cluster_manager_client.py", line 455, in create_cluster
    project_id=project_id, zone=zone, cluster=cluster, parent=parent
ValueError: Protocol message Cluster has no "workload_identity_config" field.
```

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by nimrodf1** [comment]
This is an issue both in container_v1 and in container_v1beta1 .

**Comment by busunkim96** [comment]
Hi @nimrodf1, This is the same issue as #21. The public-facing protos have not been updated for some time, so there are features that are not present in the generated libraries.

Googlers, see internal lissue 154160334.

**Comment by busunkim96** [comment]
This is partially addressed by 2.2.0, which updates the v1 surface. https://github.com/googleapis/python-container/blob/master/CHANGELOG.md#220-2020-11-17

https://github.com/googleapis/python-container/blob/20401fb2dfa4ac59d385926d345db979473c4688/google/cloud/container_v1/types/cluster_service.py#L1190

A similar update will happen for v1beta1 later this week.

**Comment by busunkim96** [comment]
Closed by #55. Changes will be available in `>=2.3.0`
